config/brakeman.ignore

```diff
@@ -1,26 +1,126 @@
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Finders::Content",
+        "method": "find_by_month"
+      },
+      "user_input": "view[:table_name]",
+      "confidence": "Weak",
+      "note": "same as line 67"
+    },
+    {
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "6bda99b671f8fb839ceebdbf40c86a186f45d5591265ec7cdf47486f97ec3f5d",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/domain/finders/aggregations.rb",
+      "line": 47,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "Arel.sql(\"SUM(#{name})\")",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Finders::Aggregations",
+        "method": "aggregations"
+      },
+      "user_input": "name",
+      "confidence": "Medium",
+      "note": "these names come from a config file, not user input"
+    },
+    {
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "c78d8ced30a8d107f9d1e33d980957eafcd971e04b770d773869d74a3718122d",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/domain/finders/content.rb",
+      "line": 66,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "scope.where(\"#{view[:table_name]}.dimensions_month_id = '#{sprintf(\"%s-%02d\", year, Date::MONTHNAMES.index(month.capitalize))}'\")",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Finders::Content",
+        "method": "find_by_month"
+      },
+      "user_input": "view[:table_name]",
+      "confidence": "Weak",
+      "note": "same as line 67"
+    },
+    {
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "da742e3eff1c5c6b7066558117c9fadfd19c0a570bf9c550229f372088eef74e",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/domain/monitor/etl.rb",
+      "line": 20,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "sum(\"facts_editions.#{edition_metric}\")",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Monitor::Etl",
+        "method": "statsd_for_edition_metrics!"
+      },
+      "user_input": "edition_metric",
       "confidence": "Medium",
-      "note": "This is used by 'guard', it's not part of the running app"
+      "note": "THis value comes from a config file, not user input."
```

Comment:
Strange capitalisaton here, is that Brakeman or something we entered in the CLI?
Nitpick if it is us, curious if it is them!?